kubectl-minio/cmd/tenant-create.go

```diff
@@ -77,6 +77,7 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	}
 	cmd = helpers.DisableHelp(cmd)
 	f := cmd.Flags()
+	f.StringVarP(&c.tenantOpts.PoolName, "pool", "p", "", "name for this pool expansion")
```

Comment:
Just checking if "name for this pool expansion" is the right help message for create tenant command

Reply:
good catch, ill fix that